roxie/udplib/udptopo.cpp

```diff
@@ -85,6 +85,33 @@ bool ChannelInfo::otherAgentHasPriority(unsigned priorityHash, unsigned otherAge
     return false;
 }
 
+static unsigned *createNewNodeHealthScore(const IpAddress &)
+{
+    return new unsigned(initIbytiDelay);
+}
+
+IpMapOf<unsigned> buddyHealth(createNewNodeHealthScore);   // For each buddy IP ever seen, maintains a score of how long I should wait for it to respond when it is the 'first responder'
+
+void noteNodeSick(const IpAddress node)
```

Comment:
const IpAddress & ?

Reply:
On modern compilers I'm not sure passing by address (for a small structure) is best.